app/models/profile.rb

```diff
@@ -149,13 +150,14 @@ def has_fraud_deactivation_reason?
 
   def in_person_verification_pending?
     # note: deactivation reason will be replaced by timestamp column
-    deactivation_reason == 'in_person_verification_pending'
+    deactivation_reason == 'in_person_verification_pending' ||
+      in_person_verification_pending_at?
```

Comment:
This is reading rather than writing. Are we ready to start reading the new value?